source/manual/access-jenkins.html.md

```diff
@@ -9,13 +9,15 @@ parent: "/manual.html"
 Our Jenkins installations ([deploy](https://deploy.integration.publishing.service.gov.uk/) and [CI](https://ci.integration.publishing.service.gov.uk/)) use GitHub for authentication and authorisation.
 
 - For integration access, you need to be added to the [GOV.UK team][]
```

Comment:
Should this now be `integration admin access`? People may wonder how integration access differs to the other environments. May also be worth adding a definition of what deploy/admin access allows you to do (perhaps just link to source/manual/rules-for-getting-production-access.html.md).

Reply:
Thanks Chris, I've made this change now.